tests/acceptance/addon-mu-smoke-test-slow.js

```diff
@@ -21,7 +21,7 @@ let addonName = 'some-cool-addon';
 let addonRoot;
 
 if (experiments.MODULE_UNIFICATION) {
-  describe('Acceptance: addon-mu-smoke-test', function() {
+  describe.skip('Acceptance: addon-mu-smoke-test', function() {
```

Comment:
just pointing out, this .skip will need to be removed before landing

Reply:
OK, I see. Thank you for explaining!  Can you create an issue (cross-linking this comment thread and linking to this `describe.skip`) so we can track fixing that separately?